pkg/apis/configuration/validation/validation.go

```diff
@@ -210,6 +210,24 @@ func validateUpstreamHealthCheck(hc *v1alpha1.HealthCheck, fieldPath *field.Path
 	return allErrs
 }
 
+func validateSessionCookie(sc *v1alpha1.SessionCookie, fieldPath *field.Path) field.ErrorList {
+	allErrs := field.ErrorList{}
+
+	if sc == nil || !sc.Enable {
```

Comment:
even if `sc.Enable` false, it is better to validate the fields.

Reply:
Just a small thing here @pleshakov @Dean-Coakley , we do it here: https://github.com/nginxinc/kubernetes-ingress/blob/master/pkg/apis/configuration/validation/validation.go#L182

For consistency I suggest either change it in HC (different PR) or leave as it is here.

Reply:
@Rulox good catch! 

I think it is better to validate in both cases. This way any problem could be discovered earlier before a hc/sc is enabled.